modules/nw-sriov-networknodepolicy-object.adoc

```diff
@@ -103,3 +106,28 @@ spec:
   linkType: ib
   isRdma: true
 ----
+
+The following example describes the configuration for an SR-IOV device in a {rh-openstack} virtual machine:
+
+.Example configuration for an SR-IOV device in a virtual machine
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: policy-sriov-net-openstack-1
+  namespace: openshift-sriov-network-operator
+spec:
+  resourceName: sriovnic1
+  nodeSelector:
+    feature.node.kubernetes.io/network-sriov.capable: "true"
+  numVfs: 1 <1>
+  nicSelector:
+    vendor: "15b3"
```

Comment:
The 'vendor' and 'deviceID' fields are mostly superfluous as there real match is to the UUID.

Reply:
@atyronesmith I don't have environment to test, if we remove vendor/device IDs from policy, will it expose virtio or other non-sriov devices as k8s resource when the given network ID matches ?

Reply:
My environment is down now as well....  However, if I remember correctly (and I look at https://github.com/openshift/sriov-network-operator/blob/f6aba07546d2c85b319bf3bfeb9deb45a2f007f5/api/v1/helper.go#L429)

A non-zero netFilter will only match that specific netfiler.

Reply:
@atyronesmith , Jason asked me to do light wordsmithing on this PR and then get it merged.  Regarding these two comments about the `nicSelector`:

Should `netFilter` be aligned with `rootDevices`?  (I'm guessing yes if the same receiver function can handle both in the helper.go file, but I'm using my imagination.)

> The 'vendor' and 'deviceID' fields are mostly superfluous as there real match is to the UUID.
Do you want the doc to suggest using `netFilter` because it is the most specific?  Depending on your answer to this question and if you indicate that `netFilter` is subordinate to `nicSelector`, then `netFilter` deserves some mention in the callout for #8, `nicSelector`.

Reply:
Hi @mikemckiernan, netFilter should be used as it is the most specific.  RootDevices does not need to be used as netFilter is specific.  NetFilter is subordinate to nicSelector and is at the same level as rootDevices, vendor, devicdeID, etc...